extensions/wikia/EmbeddableDiscussions/EmbeddableDiscussionsController.class.php

```diff
@@ -77,19 +76,18 @@ private function checkArguments( array $args, $modelData, &$errorMessage ) {
 				intval( $columns ) > self::COLUMNS_MAX ||
 				intval( $columns ) < self::COLUMNS_MIN
 			) {
-				// TODO: Refactor error messages to avoid direct concatenation, see JPN-657
-				$errorMessage = wfMessage( 'embeddable-discussions-parameter-error', 'columns' )->plain() .
-					wfMessage( 'embeddable-discussions-parameter-error-range', self::COLUMNS_MIN , self::COLUMNS_MAX )->plain();
+				$errorMessage = wfMessage( 'embeddable-discussions-parameter-error', 'columns',
+						wfMessage( 'embeddable-discussions-parameter-error-range',
+							self::COLUMNS_MIN , self::COLUMNS_MAX )->plain() )->plain();
 
 				return false;
 			}
 		}
 
 		// category must be a valid category
 		if ( $modelData['invalidCategory'] ) {
-			// TODO: Refactor error messages to avoid direct concatenation, see JPN-657
-			$errorMessage = wfMessage( 'embeddable-discussions-parameter-error', $args['category'] )->plain() .
-				wfMessage( 'embeddable-discussions-parameter-error-category' )->plain();
+			$errorMessage = wfMessage( 'embeddable-discussions-parameter-error', $args['catid'],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "catid" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)